### ISSUE TITLE
createEducation > Getting "Could not create education" error

### ISSUE THREAD

**Issue by iPiyushPatel** [SeedCompany/cord-api-v3#1326]
**Describe the bug**
createEducation > Getting `Could not create education` error

**Steps to Reproduce**
On Graphql playground or Postman - create an API request as below snap & attached notepad file and execute the request

**Expected Result**
Education should be created.

**Actual Result**
Getting `Could not create education`  error.
See video - https://drive.google.com/file/d/1Dj3z4RuCPSKmpWl098VJ8OMp8nZlqOvw/view?usp=drivesdk

**Note:** I can replicate the same issue in unavailability, partnership module

**GraphQL Query(createEducation ):**
![image](https://user-images.githubusercontent.com/5212585/95444658-af344d80-097b-11eb-8007-ab597fa2dad9.png)

**GraphQL response**
```
{
    "errors": [
        {
            "message": "Could not create education",
            "extensions": {
                "codes": [
                    "Server"
                ],
                "status": 500,
                "code": "Server",
                "exception": {
                    "message": "Could not create education",
                    "stacktrace": [
                        "at EducationService.create (components\\user\\education\\education.service.ts:196:13)"
                    ]
                }
            },
            "locations": [
                {
                    "line": 2,
                    "column": 3
                }
            ],
            "path": [
                "createEducation"
            ]
        }
    ],

```

**Query and Input File**
[GettingCouldNotCreateEducationError.txt](https://github.com/SeedCompany/cord-api-v3/files/5347005/GettingCouldNotCreateEducationError.txt)

CordAPIV3# c35e1e5a

**Comment by zkhin** [comment]
This ticket could be closed after #1329  is merged

**Comment by iPiyushPatel** [comment]
@zkhin  
Verified and working fine on the latest master a35fe809.
